bot/cogs/snakes.py

```diff
@@ -15,35 +17,70 @@ class Snakes:
-        """
-        Go online and fetch information about a snake
-
-        The information includes the name of the snake, a picture of the snake, and various other pieces of info.
-        What information you get for the snake is up to you. Be creative!
-
-        If "python" is given as the snake name, you should return information about the programming language, but with
-        all the information you'd provide for a real snake. Try to have some fun with this!
-
-        :param name: Optional, the name of the snake to get information for - omit for a random snake
-        :return: A dict containing information on a snake
-        """
+    async def is_snek(self, name: str = None) -> Dict[str, Any]:
+        if name in WikiListener.get_all_snek():
+            return True
+        else:
+            return False
 
     @command()
     async def get(self, ctx: Context, name: str = None):
-        """
-        Go online and fetch information about a snake
+        if name is None:
+            ctx.send("Ensure your command specifies the correct arguments.")
+        state = await self.is_snek(name)
+        if state:
+            if WikiListener.get_snek_scientific(name) is None:
+                title = await WikiListener.get_snek_scientific(name)
+                description = await WikiListener.get_snek_description(name)
+                embed = discord.Embed(title=f"{name}", description=f"{description}", color=0x00ff80)
+                try:
+                    thumbnail = await WikiListener.get_snek_thumbnail(name)
+                    embed.set_thumbnail(url=f'{thumbnail}')
+                except aiohttp.HTTPException:
+                    pass
+                await ctx.send(embed=embed)
+            else:
+                title = await WikiListener.get_snek_scientific(name)
+                description = await WikiListener.get_snek_description(name)
+                embed = discord.Embed(title=f"{name}", description=f"{description}", color=0x00ff80)
+                embed.set_author(name=f"{title}")
+                try:
+                    thumbnail = await WikiListener.get_snek_thumbnail(name)
+                    embed.set_thumbnail(url=f'{thumbnail}')
+                except aiohttp.HTTPException:
+                    pass
+                await ctx.send(embed=embed)
+        else:
+            await ctx.channel.send(f"Did not find {name} in the database.")
+
+    @command()
+    async def quiz(self, ctx: Context, name: str = None):
+        async def addmoji(msg, emojilist):
+            for emoji in emojilist:
+                await msg.add_reaction(emoji)
 
-        This should make use of your `get_snek` method, using it to get information about a snake. This information
-        should be sent back to Discord in an embed.
+        quemoji = ['🇦', '🇧', '🇨', '🇩']
+        question = ['Which snek is the sneakiest snek?', '🇦Cobra\n\n🇧Regular Snek\n\n🇨Python\n\n🇩<Cobra></Cobra>', '🇨']
```

Comment:
Any reason for Cobra being in `<` and `>`?

Reply:
No reason at all. I am still testing the features the questions will be in a seperate file. Probably json.